### DISCUSSION TITLE
Good hosting for postgres?

### DISCUSSION THREAD

**Discussion by mbahArip** [forscht/ddrive#60] [Q&A] [answered]
Is there any recommended service for hosting postgresql?

**Comment by forscht** [comment, marked as answer]
You can try using https://neon.tech/ It took me under 5 minutes to sign up with GitHub and get it running.

**Reply by mbahArip** [reply]
Thank you very much!

**Comment by forscht** [comment]
Other alternatives - https://hasura.io/blog/top-postgresql-database-free-tier-solutions/
